src/config/cacheConfig.ts

```diff
@@ -6,8 +6,10 @@ dotenv.config();
 verifyCriticalEnvVariable('CACHE_HOST');
 verifyCriticalEnvVariable('CACHE_PORT');
 
-export const cacheConfig = {
-  host: process.env.CACHE_HOST,
-  port: Number.parseInt(process.env.CACHE_PORT),
-  connectTimeout: Number.parseInt(process.env.CACHE_CONNECT_TIMEOUT) || 30000, // 30 seconds
-};
+const host = process.env.CACHE_HOST;
+const port = Number.parseInt(process.env.CACHE_PORT);
+const connectTimeout = Number.parseInt(process.env.CACHE_CONNECT_TIMEOUT) || 30000; // 30 seconds
+
+export const cacheConfig = { host, port, connectTimeout };
+
+export const cacheTLS = `rediss://${host}:${port}`;
```

Comment:
L15: Just checking that it's "rediss" with two "s"es.

Also, is it a good idea to mention that we use TLS for cache when not in development mode somewhere in the README?

Reply:
Oh, and also, should you update the CHANGELOG?

Reply:
Yeah, was going to hold off on all that until I know these changes will work